src/main/java/hu/dpc/phee/operator/streams/RecordParser.java

```diff
@@ -358,13 +358,19 @@ private void processJsonObject(JsonNode jsonNode, TransferTransformerConfig.Tran
     }
 
     @Transactional
-    public void parseSubBatchDetails(String jsonString) throws JsonProcessingException {
+    public void parseSubBatchDetails(String jsonString, String bpmn, Long timestamp) throws JsonProcessingException {
         ObjectMapper objectMapper = new ObjectMapper();
         List<Batch> batches = Arrays.asList(objectMapper.readValue(jsonString, Batch[].class));
 
         for (Batch bt : batches) {
             Optional<Batch> existingBatchOpt = batchRepository.findBySubBatchId(bt.getSubBatchId());
 
+            Optional<TransferTransformerConfig.Flow> config = transferTransformerConfig.findFlow(bpmn);
+            if(!config.get().getName().contains("bulk_processor")) {
```

Comment:
What if there are other workflows that are having this word bulk processor.

Reply:
@fynmanoj The bpmn name having bulk_processor is the parent( or batches product flow), so if in future if we have any more bpmn names with bulk processor it will be for parent flow only. Here in this scenario we need to exclude these cases and only enter data into transfers table when the type is batches and the flow is a child(debulked/implementation flow based on payment mode).

Reply:
If this scenario is specific to workflow, can't we use the the full workflow name ie bulk_processor_dfspid. We need to use available dfsp IDs instead of the placeholder dfspid.
The idea is to make this line specific and to avoid technical debt for future enhancements